ghcide/src/Development/IDE/Plugin/CodeAction/ExactPrint.hs

```diff
@@ -50,22 +49,15 @@ data Rewrite where
 -- | Convert a 'Rewrite' into a 'WorkspaceEdit'.
 rewriteToEdit ::
   DynFlags ->
-  Uri ->
   Anns ->
   Rewrite ->
-  Either String WorkspaceEdit
-rewriteToEdit dflags uri anns (Rewrite dst f) = do
+  Either String [TextEdit]
+rewriteToEdit dflags anns (Rewrite dst f) = do
   (ast, (anns, _), _) <- runTransformT anns $ f dflags
```

Comment:
So, your suggestion in #1274  is to apply `setEntryDPT _ dp00` to the result of `f dflags` before passing ast value to `runTransformT`, right?

Reply:
Yes, that's right